en/sim_gazebo_gz/README.md

```diff
@@ -31,16 +31,42 @@ sudo apt-get install gz-garden
 
 ## Running the Simulation
 
-Gazebo SITL simulation can be conveniently run through a `make` command as shown below:
+Gazebo SITL simulation can be conveniently run using a `make` command as shown below:
 
 ```sh
 cd /path/to/PX4-Autopilot
 make px4_sitl gz_x500
 ```
 
-This will run both the PX4 SITL instance and the Gazebo client.
+This runs both the PX4 SITL instance and the Gazebo client.
 Note that all gazebo make targets have the prefix `gz_`.
 
+Another way that Gazebo SITL can be connected is in _standalone mode_.
+In standalone mode PX4 SITL and Gazebo are started separately in their own terminals.
+You start PX4 in standalone mode by prefixing the make command with `GZ_PX4_STANDALONE=1`:
+
+```sh
+cd /path/to/PX4-Autopilot
+GZ_PX4_STANDALONE=1 make px4_sitl gz_x500
+```
+
+If you have not yet started _gz-server_ when you run the `make` command, you will see the following warning until gazebo has been started and an instance of _gz-server_ is detected by PX4:
+
+```sh
+WARN [gz bridge] Service call timed out as Gazebo has not been detected
+```
+
+The simplest way to start the simulation is to run the `simulation-gazebo` script found in the [PX4-gazebo-models](https://github.com/PX4/PX4-gazebo-models) repository.
+This will launch a _gz-server_ instance that can be started with any world and vehicle.
+For more information and arguments, see [Gazebo Models](./gazebo-models.md).
```

Comment:
So by default it just starts with the grey plane world right? What vehicle does it start with by default? 

I think you're basically saying that this script is what feeds GZ with the environment variables it needs to choose the right model and world.

Reply:
Grey plane is always default. There is no vehicle default (same way there is nothing to see when you run just `make px4_sitl`). This script will just launch the world with the necessary environmental variables (namely `GZ_PARTITION` and `GZ_IP`) and PX4 will then connect and provide the name of the desired vehicle. gz-server will then go look for that model in its own resource paths (found in `/.simulation-gazebo` per default) and load it into the world. 
Additional point that is not mentioned at the moment: If you want to run PX4 and GZ across different hosts, you will need to set `GZ_PARTITION` and `GZ_IP` on the PX4 host, as well as `GZ_RELAY`. At the moment, this is not mentioned on the user-guide but it should probably be mentioned, because this can be tricky to work out on your own. You can find an overview of the variables and their definitions here: https://gazebosim.org/api/transport/12/envvars.html

Reply:
>  If you want to run PX4 and GZ across different hosts, you will need to set GZ_PARTITION and GZ_IP on the PX4 host, as well as GZ_RELAY. At the moment, this is not mentioned on the user-guide but it should probably be mentioned, because this can be tricky to work out on your own. You can find an overview of the variables and their definitions here: https://gazebosim.org/api/transport/12/envvars.html

I think it would be useful to provide some examples of this in the "Gazebo Models Repository" doc. Leaving to you to do.

Reply:
I've added a networking example that explains how to connect PX4 and gazebo on a single host and an example on how to run it on multiple hosts.